cloudcvIde/static/cloudcvIde/js/setParams.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import Field from './field';
+import data from './data';
+
+class SetParams extends React.Component {
+  constructor(props) {
+    super(props);
+    this.changeParams = this.changeParams.bind(this);
+    this.changeProps = this.changeProps.bind(this);
+    this.trainOnly = this.trainOnly.bind(this);
+  }
+  changeProps(prop, value) {
+    const net = this.props.net;
+    let layer = net[this.props.selectedLayer];
+    layer = JSON.parse(JSON.stringify(layer));
+    layer.props[prop] = value;
+    this.props.modifyLayer(layer);
+  }
```

Comment:
Rename `props` - it's difficult to distinguish between `layer.props` and ReactJS `this.props`.